polaris-react/src/components/IndexTable/IndexTable.scss

```diff
@@ -259,6 +259,48 @@ $loading-panel-height: 53px;
     }
   }
 
+  &.TableRow-child {
+    cursor: default;
+    // stylelint-disable -- Polaris component custom properties
+    --pc-index-table-checkbox-offset-left: var(--p-space-4);
+    --pc-index-table-checkbox-width: var(--p-space-5);
+    --pc-index-table-checkbox-border-width: var(--p-border-width-2);
+    --pc-index-table-checkbox-child-offset: calc(
+      var(--pc-index-table-checkbox-offset-left) +
+        var(--pc-index-table-checkbox-border-width) +
+        var(--pc-index-table-checkbox-width)
+    );
+
+    #{$se23} & {
```

Comment:
Since summer editions will be cleaned up by the end of the week, would it make sense to rebase this branch off of the summer editions clean-up to reduce the amount of changes we are making and the number of cases we need to tophat as a result?

Reply:
+1, you'll need to rebase off of `v12.x.x`, I've updated the base branch of the PR.

Reply:
Done! Thank you!